otaclient/app/configs.py

```diff
@@ -184,6 +184,8 @@ class BaseConfig(_InternalSettings):
     # default version string to be reported in status API response
     DEFAULT_VERSION_STR = ""
 
+    DEBUG_MODE = False
+
```

Comment:
About DEBUG_MODE, when enabled, currently otaclient will behave as follow:
1. `failure_traceback` will also be reported via status API(normally only being logged).

In the future, more features like logging to file when DEBUG_MODE is on might be supported.